pyServer/GuestFishWrapper.py

```diff
@@ -205,23 +205,25 @@ def RunCredentialScanner(self, targetDir, operationOutFile):
                 secret_found = line[2]
                 line_number = line[4]
                 is_redacted = False
-
-                # Remove secret - delete line for small files, entire file for large files
-                if os.path.isfile(source_file):
-                    if os.path.getsize(filepath) < 1000000:
-                        is_redacted = True
-                        with open(source_file, "r+", encoding='utf-8') as f:
-                            lines = f.readlines()
-                            f.seek(0)
-                            for i, line in enumerate(lines):
-                                if (i+1) != int(line_number):
-                                    f.write(line)
-                                else:
-                                    f.write("***REDACTED***")
-                            f.truncate()
-                    else:
-                        os.remove(source_file)
-
+                try:
+                    # Remove secret - delete line for small files, entire file for large files
+                    if os.path.isfile(source_file):
+                        if os.path.getsize(source_file) < 1000000:
+                            is_redacted = True
+                            with open(source_file, "r+", encoding='utf-8') as f:
+                                lines = f.readlines()
+                                f.seek(0)
+                                for i, line in enumerate(lines):
+                                    if (i+1) != int(line_number):
+                                        f.write(line)
+                                    else:
+                                        f.write("***REDACTED***")
+                                f.truncate()
+                        else:
+                            os.remove(source_file)
+                except:
```

Comment:
This should be of type `OSError`, right?  Perhaps we can catch that type and log the actual error being thrown?

Reply:
Yes, that's the good idea. Let's catch the type and log the actual error. Which will me more specific then just handling OSError

Reply:
I see you're capturing the base Exception type and printing the exception now. Do you know when there's a file related error, what the stringified output will be? Will this include the file name, etc? 

The benefit of catching OSError is you have more specific properties to use for message output. See: https://docs.python.org/3/library/exceptions.html#OSError

Reply:
For file related error, sample message will be like below
"CredentialScanner: Failed to run due to [Error] No such file or directory: 'filename'